src/NosnocNLP.m

```diff
@@ -110,7 +110,8 @@
             obj.sigma_p = sigma_p;
             obj.p = [sigma_p;mpcc.p];
 
-            obj.cost = mpcc.cost;            
+            obj.cost = mpcc.cost;
+            obj.objective = mpcc.objective;
```

Comment:
I am a bit confused about the difference between cost and objective.
I suppose one is the pure NLP cost function, and the other has the algorithmic parts like step equilibration, comp. residual and so on. Add a comment to explain the difference or provide an explicit name such as
`objective` and`objective_with_algoritmic_terms`